conftest.py

```diff
@@ -8,13 +8,19 @@
 from config.settings import DATABASES
 
 TEST_DB = "mathesar_db_test_database"
```

Comment:
@eito-fis Once you merge `master` in, I think it would be good to read these from the `python-decouple` instead of defining them in two places.